docs/source/markdown/podman-manifest-push.1.md.in

```diff
@@ -14,6 +14,15 @@ The list image's ID and the digest of the image's manifest.
 
 ## OPTIONS
 
+#### **--add-compression**=*compression*
+
+Makes sure that requested compression variant for each platform is added to the manifest list keeping original instance
+intact in the same manifest list. Supported values are (`gzip`, `zstd` and `zstd:chunked`)
+
+Note: This is different than `--compression-format` which replaces the instance with requested with specified compression
+while `--add-compression` makes sure than each instance has it variant added to manifest list without modifying the
```

Comment:
For some reason I cannot propose a change via the GitHub UI but the sentence above has typos:

makes sure tha**t** each instance has **its** variant added to manifest list without modifying the
original instance.

I still find the note hard to understand and suggest to be more explicit and less abstract:

"Note that `--compression-format` controls the compression format of each instance in the manifest list.  `--add-compression` will add another variant for each instance in the list with the specified compressions.  `--compression-format gzip --add-compression zstd` will push a manifest list with each instance being compressed with gzip plus an additional variant of each instance being compressed with zstd."

I am sure there's a better description than mine but I find it important to be more explicit.

Reply:
Maybe worth mentioning that it can be specified multiple times.

Reply:
> Maybe worth mentioning that it can be specified multiple times.

Amended text and added above line as well.